src/hmr/hotModuleReplacement.js

```diff
@@ -0,0 +1,165 @@
+/* global document, window */
+/* eslint func-names: 0 */
+/* eslint no-var: 0 */
+/* eslint vars-on-top: 0 */
+/* eslint prefer-arrow-func: 0 */
+/* eslint prefer-rest-params: 0 */
+/* eslint prefer-arrow-callback: 0 */
+/* eslint prefer-template: 0 */
+
+var normalizeUrl = require('normalize-url');
```

Comment:
Normalize@3 has `const` and other ES next features https://github.com/sindresorhus/normalize-url/blob/v3.3.0/index.js#L3, will be great transpile this, but we can setup this in other PR (just copy/paste code and rewrite this on es2015)

Reply:
what PR, a pr to normalize or here? I do know that some users have had issues - i wouldn't want this to release and cause problems right away

Reply:
will downgrade to v2